OptimizelySDKCore/OptimizelySDKCore/OPTLYProjectConfig.m

```diff
@@ -182,6 +183,22 @@ - (OPTLYAttribute *)getAttributeForKey:(NSString *)attributeKey {
     return attribute;
 }
 
+- (nullable NSString *)getAttributeIdForKey:(nonnull NSString *)attributeKey {
+    OPTLYAttribute *attribute = self.attributeKeyToAttributeMap[attributeKey];
```

Comment:
Make sure, code doesn't break here. if attributekey is not in the map.

Reply:
Yes it is checked here https://github.com/optimizely/objective-c-sdk/blob/21f3a0cfa4055fd3eab43a6619c23585452dc883/OptimizelySDKCore/OptimizelySDKCore/OPTLYProjectConfig.m#L190